[needs tech review] - adding browser session time (#3760)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/about-cloud/browsers.md

To improve your experience using dbt Cloud, we suggest that you turn off ad blockers.
:::

### Browser sessions

> A session is a period of time during which you’re signed in to a dbt Cloud account from a browser. dbt Cloud will not automatically log you out after a set time. However, if you close your browser, it will end your session and log you out. You'll need to log in again the next time you try to access dbt Cloud. 

Author: ```suggestion
A session is a period of time during which you’re signed in to a dbt Cloud account from a browser. If you close your browser, it will end your session and log you out. You'll need to log in again the next time you try to access dbt Cloud. 
```

> A session is a period of time during which you’re signed in to a dbt Cloud account from a browser. dbt Cloud will not automatically log you out after a set time. However, if you close your browser, this action ends your dbt Cloud session and logs you out. You'll need to re-login the next time you try to access dbt Cloud. 

Reviewer: ```suggestion
A session is a period of time during which you’re signed in to a dbt Cloud account from a browser. dbt Cloud will not automatically log you out after a set time. However, if you close your browser, it will end your session and log you out. You'll need to log in again the next time you try to access dbt Cloud. 
```

> A session is a period of time during which you’re signed in to a dbt Cloud account from a browser. dbt Cloud will not automatically log you out after a set time. However, once you close your browser, you end your dbt Cloud session. This action logs you out and you'll need to re-login the next time you try to access dbt Cloud. 

Author: ```suggestion
A session is a period of time during which you’re signed in to a dbt Cloud account from a browser. dbt Cloud will not automatically log you out after a set time. However, if you close your browser, this action ends your dbt Cloud session and logs you out. You'll need to re-login the next time you try to access dbt Cloud. 
```

> If you've logged in using [SSO](/docs/cloud/manage-access/sso-overview) or [OAuth](/docs/cloud/manage-access/set-up-snowflake-oauth), you can customize your session time, which might vary depending on your identity provider (IdP).

Author: ```suggestion
If you've logged in using [SSO](/docs/cloud/manage-access/sso-overview) or [OAuth](/docs/cloud/git/connect-github#personally-authenticate-with-github), you can customize your session time, which might vary depending on your identity provider (IdP).
```

> If you've logged in using [SSO](/docs/cloud/manage-access/sso-overview) or [OAuth](/docs/cloud/git/connect-github#personally-authenticate-with-github), you can customize your session time, which might vary depending on your identity provider (IdP).

Author: ```suggestion
If you've logged in using [SSO](/docs/cloud/manage-access/sso-overview) or [OAuth](/docs/cloud/git/connect-github#personally-authenticate-with-github), you can customize your maximum session duration, which might vary depending on your identity provider (IdP).
```

> If you've logged in using [SSO](/docs/cloud/manage-access/sso-overview) or [OAuth](/docs/cloud/manage-access/set-up-snowflake-oauth), you can customize your session time, which may vary depending on your identity provider (IdP).

Reviewer: ```suggestion
If you've logged in using [SSO](/docs/cloud/manage-access/sso-overview) or [OAuth](/docs/cloud/manage-access/set-up-snowflake-oauth), you can customize your session time, which might vary depending on your identity provider (IdP).
```
